test/api/test_session.py

```diff
@@ -429,6 +324,20 @@ def test_check_response_raises_http_error(self, mock_requests):
             session._check_response("test_url", create_mock_response(400))
         self.assertEqual(str(err.exception), "Test error 400")
 
+    def test_check_response_calls_error_message_func(self, mock_requests):
```

Comment:
I see you've added them elsewhere - this test is missing the SETUP/CALL/ASSERT comments